zmon-controller-app/src/main/java/org/zalando/zmon/ZmonApplication.java

```diff
@@ -12,6 +19,21 @@
 @SpringBootApplication
 @ComponentScan(basePackages = {"org.zalando.zmon"})
 public class ZmonApplication {
+	@Bean
+	public io.opentracing.Tracer jaegerTracer() {
+
+		Configuration config = Configuration.fromEnv();
+
+		io.opentracing.Tracer tracer = config.getTracer();
+
+    	/*return new Configuration("zmon-controller", new Configuration.SamplerConfiguration(ProbabilisticSampler.TYPE, 1),
```

Comment:
commented code generally should be deleted

Reply:
done